api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -75,31 +75,28 @@
     @Mock
     EligibilityAndEntitlementDecisionFactory eligibilityAndEntitlementDecisionFactory;
 
-    @ParameterizedTest
-    @MethodSource("allEligibilityOutcomeValues")
-    void shouldReturnDuplicateWhenLiveClaimAlreadyExists(EligibilityOutcome eligibilityOverrideOutcome) {
+    @Test
+    void shouldReturnDuplicateWhenLiveClaimAlreadyExistsWithoutEligibilityOverride() {
+        shouldReturnDuplicateWhenLiveClaimAlreadyExists(null);
+    }
+
+    @Test
+    void shouldReturnDuplicateWhenLiveClaimAlreadyExistsWithEligibilityOverride() {
+        aConfirmedEligibilityOverrideWithNoChildren();
+    }
+
+    private void shouldReturnDuplicateWhenLiveClaimAlreadyExists(EligibilityOverride eligibilityOverride) {
```

Comment:
This is only used in one test, so this would be better inlined, especially as the only value used for EligibilityOverride is null.

Reply:
One test with null and other with valid eligibility override object, Comment above should resolve this